assets/css/v2/lcd_common/departures/later_departures.scss

```diff
@@ -0,0 +1,83 @@
+@use "sass:math";
+
+.later-departures {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-height: 280px;
+  background-color: $warm-neutral-90;
+
+  &__header {
+    box-sizing: inherit;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin: 0 32px;
+    border-bottom: 4px solid $true-grey-70;
+
+    h3 {
+      box-sizing: inherit;
+      flex-shrink: 0;
+      padding: 18px 0;
+      margin: 0;
+      font-family: Inter;
+      font-size: 40px;
+      font-weight: 500;
+      line-height: 43px;
+      color: $cool-black-15;
+      text-transform: uppercase;
+      letter-spacing: 2px;
+    }
+
+    ol {
+      display: flex;
+      flex-direction: row;
+      gap: 24px;
+      justify-content: space-between;
+      list-style: none;
+
+      .route-pill {
+        position: static;
+        box-sizing: border-box;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        width: auto;
+        min-width: 86px;
+        height: 44px;
+      }
+
+      .route-pill--yellow {
+        box-shadow: 1px 1px 1px 0 rgba(0 0 0 / 25%);
+      }
+
+      .route-pill__text {
+        position: static;
```

Comment:
Curious what the story behind this line is.

Reply:
I had to change the layout for the smaller route pills so they could expand based on their contents.

The existing route pill component uses `position: relative` and `position: absolute` hacks to achieve horizontal and vertical centering of their text. This relied on fixed route pill widths. `position: static` resets position to the default content flow. Then I use flexbox to get the same overall centering.

Reply:
Would it be feasible to backport your flexbox improvements to route pills outside this context? If not that's fine, though I think we should add a comment here explaining what's going on (that this is overriding a value used to do centering on route pills elsewhere, so the centering can be done in a different way).

Reply:
We can totally get all of the route pills using flexbox instead but without an easy way to verify every possible route pill still looks correct I'm not comfortable making those changes. 📌 

This would be a great opportunity for Storybook or something similar.

Do we keep track of "tech debt" or "improvement" tasks any differently than other kinds of tasks in Asana?

Reply:
I've added a comment explaining this code for now 32ca43f3